src/java.base/share/classes/java/util/Comparator.java

```diff
@@ -189,6 +188,44 @@ default Comparator<T> reversed() {
         return Collections.reverseOrder(this);
     }
 
+    /**
+     * Returns the greater of two values according to this comparator.
+     * If the arguments are equal with respect to this comparator,
+     * the {@code a} argument is returned.
+     *
+     * @param   a   an argument.
+     * @param   b   another argument.
+     * @param   <U> the type of the arguments and the result.
+     * @return  the larger of {@code a} and {@code b} according to this comparator.
+     * @throws  ClassCastException if the collection contains elements that are
```

Comment:
What 'collection' meaned here?

Reply:
Good catch, thank you. The description was copied from `Collections::max`, which is not so appropriate here. I think it's better to copy it from `Comparator::compare` (also, added `@throws NPE` for consistency).